test/runtime/runtime-go/Test_Go/Test_Address.go

```diff
@@ -0,0 +1,58 @@
+package Test_Go
+
+import (
+	"encoding/json"
+	runtimego "runtimego/target"
+	"testing"
+)
+
+func TestShouldbeAbleToSerializeModel(t *testing.T) {
```

Comment:
We still need to compare the actual serialized value with something we expect, you can grab the one in the other runtime test 🙂

Reply:
Yep, thats about it 👍